pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/controversial/xml/UnnecessaryParentheses.xml

```diff
@@ -1,88 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<test-data>
-    <test-code>
```

Comment:
There's already [this one](https://github.com/pmd/pmd/pull/554/files/69b32d4c0027885d1458f7a0bd9e55e37a2b3f52..2d5f1355c2fde0ed3eb12c99def8168ec4294c14#diff-35a3759c1754926dbc7d99dc1743033bR418) so I thought I could remove it